qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -4,6 +4,59 @@
 from qcodes import VisaInstrument
 from qcodes.instrument.parameter import ArrayParameter
 from qcodes.utils.validators import Numbers, Ints, Enum, Strings
+from qcodes.instrument.channel import InstrumentChannel, ChannelList
+
+
+class SR830AuxArray(InstrumentChannel):  # We see that InstrumentChannel, ChannelList are something of a misnomer
```

Comment:
I would call this SR830AuxChannel because thats what it is

Reply:
Ok